src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.NETStandardTargetFrameworks.targets

```diff
@@ -0,0 +1,34 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.NETStandardTargetFrameworks.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <!-- Default the check to true, but allow developers to turn the warning off. -->
+    <CheckNotRecommendedTargetFramework Condition="'$(CheckNotRecommendedTargetFramework)' == '' and '$(OutputType)' == 'Library'">true</CheckNotRecommendedTargetFramework>
```

Comment:
Do we want / need the `$(OutputType)` check? When targeting netstandard1.x you most likely are building a library but it wouldn't harm to still emit the warning if someone tries to target netstandard1.x and builds an executable.

Reply:
I thought we already stopped supporting building a .NET Standard .exe anyways. But it's probably easiest to not check output type.

Reply:
Yeah, I thought about that originally but figured there was no harm in including the warning for exes

Reply:
```suggestion
    <CheckNotRecommendedTargetFramework Condition="'$(CheckNotRecommendedTargetFramework)' == ''">true</CheckNotRecommendedTargetFramework>
```